docs/source/tutorials/qmd/07-groups.qmd

```diff
@@ -0,0 +1,330 @@
+---
+engine: knitr
+---
+
+# Advanced Group Usage
+
+This is a tutorial on how Liesel's {class}`.Group` class can be used to derive
+new classes that represent re-usable model parts. This tutorial covers the
+following parts:
+
+1. Derive new classes from {class}`.Group` to represent related {class}`.Var`s.
+2. Use these groups to create a Gibbs kernel.
+
+
+## Background
+
+As the topic for this group tutorial, we will create groups for a semiparametric
```

Comment:
i would also briefly discuss that the spline is used to approximate $f(x)$ as $X \beta$ and what the purpose of the prior on $tau$ is. just two sentences. later $X$ and $X \beta$ are used but the reader might not be familiar.

Reply:
I'll do that.